audition/distance_from_best.py

```diff
@@ -225,7 +239,7 @@ def generate_plot_data(
                 )
                 SELECT dist.model_group_id, pct_diff, mg.model_type,
                        COUNT(*) AS num_models,
-                       AVG(CASE WHEN below_best <= pct_diff THEN 1 ELSE 0 END) AS pct_of_time
+                       AVG(CASE WHEN dist_from_best_case <= pct_diff THEN 1 ELSE 0 END) AS pct_of_time
```

Comment:
The thing I'm struggling with is that it sort of depends on the scale. For 0-1 metrics like precision and recall, graphing the full range seems a lot cleaner since keeping the graphs on a consistent set of axes definitely helps make them more interpretable. However, for metrics on a larger scale, that could include a lot of unnecessary dead space and for unbounded metrics simply isn't meaningful at all. But I also kind of hesitate to make the decorator too cumbersome by requiring too much to be specified there...

Maybe a compromise is to check if the values are all between 0 and 1 and use those bounds in that case otherwise use the max/min from the data?